app/main/controllers/template/RTMediaNav.php

```diff
@@ -240,7 +240,7 @@ public function admin_nav() {
 	 * Add submenus for My account.
 	 */
 	public function sub_nav() {
-		global $rtmedia, $rtmedia_query;
+		global $rtmedia, $rtmedia_query, $wp;
```

Comment:
I think you can remove the `$wp` global variable from here. Since we are not checking anything using global `$wp`

Reply:
Done